specification/agfood/resource-manager/readme.go.md

```diff
@@ -0,0 +1,26 @@
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+```yaml $(go)
+go:
+  license-header: MICROSOFT_APACHE_NO_VERSION
```

Comment:
azure sdk for go is using MIT license now
```suggestion
  license-header: MICROSOFT_MIT_NO_VERSION
```

Reply:
Updated